pokemongo_bot/cell_workers/follow_cluster.py

```diff
@@ -42,7 +42,7 @@ def work(self):
 
                 log_str = log_lure_avail_str + 'Move to destiny. ' + str(cnt) + ' ' + log_lured_str + \
                           'pokestops will be in range of ' + str(self.radius) + 'm. Arrive in ' \
-                          + str(distance(self.bot.position[0], self.bot.position[1], lat, lng)) + 'm.'
+                          + str(round(distance(self.bot.position[0], self.bot.position[1], lat, lng))) + 'm.'
```

Comment:
Agreed, I'm not sure how to separate the commits into different PRs yet. Feel free to drop this commit from the PR.